### ISSUE TITLE
Give a list of examples provided for users 

### ISSUE THREAD

**Issue by Cletip** [Overdr0ne/holymotion#1]
Not really an issue, but rather an improvement :)
I think it would be very useful to put examples somewhere (example mean block of code to apply the famous "copy-paste-work" method.) Maybe exactly like org-ql : 
https://github.com/alphapapa/org-ql/blob/master/examples.org

**Comment by Overdr0ne** [comment]
I agree, seeing as I've built it as a library, good to show some of the variations.

**Comment by Cletip** [comment]
Perfect, close this issue whenever you want :)
